### ISSUE TITLE
Gold, Silver and Copper

### ISSUE THREAD

**Issue by Gridomin** [cstby/foundryvtt-world-currency-5e#7]
Hi cstby,
I love your module. Thank you very much.
But I have problem only using the three "standard" coins gold, silver and copper. So when I hide Platinum and Electrum, those are not seen on the character sheets. But I have the feeling, that the system is still working with them. I use Merchant Sheets and LootSheets. 
After me PCs trade with merchants and want to clear their coin inventory by clicking "convert", all coins are converted to platinum, so they disappear from the sheet.
I am doing something wrong? I don't get it, sorry.
Thanx for your help!

**Comment by cstby** [comment]
That's correct. LootSheets is not compatible. They way that it "makes change" assumes that all five currencies are used at the normal conversion rate. That module itself would need to change to better support custom currenices.

Merchant Sheets should have full compatibility. If you use Merchant Sheets and this module at the same time, the merchant will ONLY use the standard currency. You can change the standard currency on the fly in the settings.
